docs/configuration.html

```diff
@@ -33,7 +33,7 @@
 
   <p>More details about broker configuration can be found in the scala class <code>kafka.server.KafkaConfig</code>.</p>
 
-  <a id="topic-config" href="#topic-config">Topic-level configuration</a>
+  <h3><a id="topic-config" href="#topic-config">3.2 Topic-Level Configs</a></h3>
```

Comment:
We should change the reference to be consistent with the other ones (i.e. no hyphen and plural).